composer.json

```
@@ -86,7 +86,7 @@
   "require": {
     "php": "^8.0",
     "ext-json": "*",
-    "dachcom-digital/emailizr": "^2.0.0",
+    "dachcom-digital/emailizr": "^2.0.0 || ^3.0.0",
```

This is enough and also more consistent with the rest:
```suggestion
    "dachcom-digital/emailizr": "^2.0 || ^3.0",
```